requirements.txt

```diff
@@ -0,0 +1,4 @@
+mido==1.2.9
```

Comment:
Oh man I love this `requirements` file approach! So much cleaner.

Reply:
You can create it with

pip3 freeze > requirements.txt

I am using a venv, so only packages needed by this script will be included.